src/clinical/clinical-api.ts

```diff
@@ -37,6 +38,28 @@ class ClinicalController {
       .status(200)
       .send(id);
   }
+
+  @HasFullWriteAccess()
+  async patchDonorCompletenessStats(req: Request, res: Response) {
```

Comment:
just a question not related to code, if we update the same donor these manual values will be overridden, right ?

Reply:
yea, that's how it is setup